dcrwallet.go

```diff
@@ -242,6 +243,17 @@ func run(ctx context.Context) error {
 			passphrase = startPromptPass(ctx, w)
 		}
 
+		// Process VSP tickets that failed to be processed.
```

Comment:
I feel iffy about putting all of this into the application startup code.  At least for decrediton's purposes, it would be fine to only report status and react to it over RPC.